app/create_app.py

```diff
@@ -71,7 +76,7 @@ def inject_global_constants():
         return dict(
             stage="beta",
             service_title=(
-                "Apply for funding to save an asset in your community"
+                gettext(u'Apply for funding to save an asset in your community')
```

Comment:
Think this `u` prefix is redundant and can be safely removed.

Reply:
☝️ I agree with this. The 'u' prefix means that a string should be interpreted as a unicode string, and 'b' means to represent the string as raw bytes in the backend - and python 3 uses unicode by default. 

Fun fact: This means that `b"😳"` would actually raise an error, since their is no bytes/ASCII representation of 😳.

Reply:
@SentientHousePlant you're my favorite